### PR TITLE
[SUB-3315] Fix example and schema

### PR DESCRIPTION
**Pull request by Jacek-202**
Fix previewAllocations

reference: https://maxioevolution.atlassian.net/browse/SUB-3315

### CODE REVIEW

File: components/schemas/Preview-Allocation-Request.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+title: Create Allocation Request
+type: object
+properties:
+  allocations:
+    $ref: "./Create-Allocation.yaml"
```
Comment (maciej-nedza) [inline components/schemas/Preview-Allocation-Request.yaml:5]: Shouldn't it be a list?

File: components/schemas/Preview-Allocation-Request.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+title: Create Allocation Request
```
Comment (maciej-nedza) [inline components/schemas/Preview-Allocation-Request.yaml:1]: Preview Allocation Request

Comment (Jacek-202) [reply components/schemas/Preview-Allocation-Request.yaml]: Preview Allocations Request

**Review by maciej-nedza — APPROVED** [review]
LGTM

**Pull request merged**